Lib/http/cookiejar.py

```diff
@@ -533,14 +533,20 @@ def parse_ns_headers(ns_headers):
 
 
 IPV4_RE = re.compile(r"\.\d+$", re.ASCII)
+def is_ip(text):
+    """Return True if text is a valid IP address."""
+    from ipaddress import ip_address
+    try:
+        ip_address(text)
```

Comment:
Now that I think about, `ip_address('[::1]')` will return False. So I don't think it'll be sufficient here.